src/index.html

```diff
@@ -11,11 +11,11 @@
   <body>
     <header>
       <h1><span>fresh</span>Picks<img src="./images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
-      <nav id="search" role="search">
-        <input id="search-input" type="text" autocomplete="off" placeholder="Search Recipes..."></input>
-        <label for="search-input" class="search-label">Search</label>
+      <form id="search" role="search">
```

Comment:
Great catch everyone. Good to remember that search bars and buttons belong with form and label tags